kinto/tests/test_views_groups.py

```diff
@@ -187,16 +187,18 @@ def setUp(self):
         super(InvalidGroupTest, self).setUp()
         self.create_bucket('beers')
 
-    def test_groups_must_have_members_attribute(self):
+    def test_groups_data_is_required(self):
```

Comment:
This means that we cannot change group permissions if we do not provide also the data with PUT.
But is it working fine with PATCH?